azure-pipelines.yml

```diff
@@ -259,6 +259,7 @@ jobs:
 - job: otbn_standalone_tests
   displayName: Run OTBN Smoke Test
   dependsOn: lint
+  condition: and(succeeded(), eq(dependencies.lint.outputs['DetermineBuildType.onlyCdcChanges'], '0'))
```

Comment:
@GregAC Just added this to skip OTBN. Other changes are amended to the pre `condition:` field.